File: personal_expenses_control_app/node/conexion.js
const mysql = require('mysql2')

const conexion = mysql.createConnection({
    host: "localhost",
    user: "jefe21054",
    password: "admin",
    database: "gastos",
})

conexion.connect((err, conn)=>{
    if (err) {
        console.log('Ha ocurrido un error al conectarse a la base de datos: ' + err)
    }else{
        console.log('Conexion exitosa')
        return conn    
    }
})

module.exports = conexion

File: library-app/conexion.py
import mysql.connector as mysql

def conectar():
    try:
        conexion = mysql.connect(
            host = 'localhost',
            user = 'jefe21054',
            password = 'admin',
            database = 'libros',
        )
        print('Se ha conectado a la base de datos')
        return conexion
    except mysql.Error as err:
        print('Ha ocurrido un error',err)
